src/rules/converters/jsdoc-format.ts

```diff
@@ -0,0 +1,18 @@
+import { RuleConverter, ConvertedRuleChanges } from "../converter";
+
+export const convertJSDocFormat: RuleConverter = () => {
+    const ruleNames: string[] = [
+        "jsdoc/check-alignment",
+        "jsdoc/require-jsdoc",
```

Comment:
Ehh, `require-jsdoc` is more of a [TSLint `completed-docs`](https://palantir.github.io/tslint/rules/completed-docs) equivalent, right? That doesn't seem like it belongs here.

Reply:
I wasn't sure which ones shall be included based on the [eslint-plugin-jsdoc](https://github.com/gajus/eslint-plugin-jsdoc) documentation as many of them doesn't satisfy the above rule behaviours.

Reply:
Yeah there's a lot of mismatch here...

> TSLint: each line contains an asterisk and asterisks must be aligned

ESLint's `check-alignment` looks similar. 

> TSLint: each asterisk must be followed by either a space or a newline (except for the first and the last)

ESLint: `check-indentation` looks like it covers this? I haven't checked too deeply.

> TSLint: the only characters before the asterisk on each line must be whitespace characters

ESLint: `check-indentation` looks like it covers this too? I haven't checked too deeply.

> TSLint: one line comments must start with /** and end with */
> TSLint: multiline comments don’t allow text after /** in the first line (with option "check-multiline-start")

I didn't see any equivalents. Wamp wump.

For the ones with equivalents, I'm happy to assume they're roughly the same, and defer trying to describe the teeny differences until an issue is filed.

For the missing ones, adding to `notices` seems reasonable.

Reply:
Alright, I will take a look at each one and provide a notice in case it doesn't satisfy at all the rule options. Still need to find some free time these days 😢